src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostMemberTyping.scala

```diff
@@ -87,6 +88,11 @@ trait GhostMemberTyping extends BaseTyping { this: TypeInfoImpl =>
     }
   }
 
+  private def hasOldExpression(n: PExpression): Messages = {
```

Comment:
2 comments:
- the name is a bit misleading as we check that is does not have old exprs
- since the error message is specific to postconditions, I'd make the entire spec input to this function
(- do we check that preconditions & termination measures are free of olds?)

Reply:
Good point, I added the missing checks for pres and termination measures